### PR TITLE
Tutorial updates for xrpl-py2 library (post-IAv3)

### PR DESCRIPTION
**Pull request by DennisDawson**
These are the updates to the Python examples to use the xrpl-py2 library.

### CODE REVIEW

File: content/tutorials/quickstart/py-batch-minting.md [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+from xrpl.clients import JsonRpcClient
+from xrpl.wallet import Wallet
+from xrpl.models.requests import AccountNFTs
+
+testnet_url = "https://s.altnet.rippletest.net:51234"
+```
+
+## Batch Mint
+
+Pass the values _seed_, _uri_, _flags_, _transfer___fee_, _taxon_, and _count_.
```
Comment (amarantha-k) [inline content/tutorials/quickstart/py-batch-minting.md]: Shouldn't these be in code font, like `seed`, `uri`, and so on? 
This was noted in the original PR as a review comment and may have missed being ported. Please take a look.

File: content/tutorials/quickstart/py-batch-minting.md [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+
+```python
+    ticket_numbers_req=xrpl.models.requests.AccountObjects(
+        account=wallet.address,
+        type='ticket'
+    )
+    ticket_response=client.request(ticket_numbers_req)
+```
+
+Create the _tickets_ variable to store an array of tickets.
```
Comment (amarantha-k) [inline content/tutorials/quickstart/py-batch-minting.md]: Similar to the comment above, _tickets_ should be `tickets`.

**Review by amarantha-k — CHANGES_REQUESTED** [review]

File: content/tutorials/quickstart/py-batch-minting.md (hunk 1 above)
Comment (DennisDawson) [reply content/tutorials/quickstart/py-batch-minting.md]: Actually, I didn't fight it before, and I won't fight it now, but here we're talking about variable as opposed to parameters, so I would personally use italics. But as long as it's consistent, I won't die on this hill.

**Review by amarantha-k — APPROVED** [review]
lgtm

**Pull request merged**